routes/events.js

```diff
@@ -5,7 +5,7 @@ const eventsCtrl = require("../controllers/events");
 const authenticateJWT = require("../auth/auth");
 
 // Event Routes
-router.get("/new", eventsCtrl.newEvents);
+router.get("/new", authenticateJWT, eventsCtrl.newEvents);
```

Comment:
so this is that functionality we talked about that you can only touch the user models if you're logged in, otherwise you can just see events and see events only

Reply:
I think we will need a separate function for non logged in users then, too much of how this works is dependent on there being a user to reference.